src/Annotations/AppKernel.php

```diff
@@ -0,0 +1,292 @@
+<?php
+
+namespace eLife\Annotations;
+
+use Aws\Sqs\SqsClient;
+use Doctrine\Common\Cache\FilesystemCache;
+use eLife\Annotations\Provider\QueueCommandsProvider;
+use eLife\ApiClient\HttpClient\BatchingHttpClient;
+use eLife\ApiClient\HttpClient\Guzzle6HttpClient;
+use eLife\ApiClient\HttpClient\NotifyingHttpClient;
+use eLife\ApiProblem\Silex\ApiProblemProvider;
+use eLife\ApiSdk\ApiSdk;
+use eLife\Bus\Limit\CompositeLimit;
+use eLife\Bus\Limit\LoggingLimit;
+use eLife\Bus\Limit\MemoryLimit;
+use eLife\Bus\Limit\SignalsLimit;
+use eLife\Bus\Queue\SqsMessageTransformer;
+use eLife\Bus\Queue\SqsWatchableQueue;
+use eLife\HypothesisClient\ApiSdk as HypothesisApiSdk;
+use eLife\HypothesisClient\Credentials\Credentials;
+use eLife\HypothesisClient\HttpClient\BatchingHttpClient as HypothesisBatchingHttpClient;
+use eLife\HypothesisClient\HttpClient\Guzzle6HttpClient as HypothesisGuzzle6HttpClient;
+use eLife\HypothesisClient\HttpClient\NotifyingHttpClient as HypothesisNotifyingHttpClient;
+use eLife\Logging\LoggingFactory;
+use eLife\Logging\Monitoring;
+use eLife\Ping\Silex\PingControllerProvider;
+use GuzzleHttp\Client;
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\Middleware;
+use Knp\Provider\ConsoleServiceProvider;
+use Monolog\Logger;
+use Pimple\Exception\UnknownIdentifierException;
+use Psr\Container\ContainerInterface;
+use Silex\Application;
+use Silex\Provider\HttpFragmentServiceProvider;
+use Silex\Provider\ServiceControllerServiceProvider;
+use Silex\Provider\TwigServiceProvider;
+use Silex\Provider\VarDumperServiceProvider;
+use Silex\Provider\WebProfilerServiceProvider;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+use Symfony\Component\HttpKernel\TerminableInterface;
+
+final class AppKernel implements ContainerInterface, HttpKernelInterface, TerminableInterface
+{
+    private $app;
+
+    public function __construct(string $environment = 'dev')
+    {
+        $configFile = __DIR__.'/../../config.php';
+        $config = array_merge(file_exists($configFile) ? require $configFile : [], require __DIR__."/../../config/{$environment}.php");
```

Comment:
as it's written now `config.php`, but `config/$environment.php` is mandatory. Either a `file_exists` is needed on the second `require` too, or empty files are needed for `config/end2end.php` and `config/continuumtest.php` (identical to `config/prod.php`)

Reply:
I have added `config/end2end.php` and `config/continuumtest.php`